app/models/form_submission_attempt.rb

```diff
@@ -26,13 +26,16 @@ class FormSubmissionAttempt < ApplicationRecord
     event :fail do
       after do
         if should_send_simple_forms_email
+          form_type = form_submission.form_type
           Rails.logger.info({
                               message: 'Preparing to send Form Submission Attempt error email',
                               form_submission_id:,
                               benefits_intake_uuid: form_submission.benefits_intake_uuid,
-                              form_type: form_submission.form_type
+                              form_type:
                             })
           simple_forms_enqueue_result_email(:error)
+          tags = ['service:veteran-facing-forms', "function:#{form_type} form submission to Lighthouse"]
+          StatsD.increment('silent_failure_avoided_no_confirmation', tags:)
```

Comment:
Are there tests you can update based on these additions? 

Possibly related: your specs have failed twice after timing out after 20 min :/